weapon-config.inc

```diff
@@ -4176,6 +4190,9 @@ static UpdateHealthBar(playerid, bool:force = false)
 
 static SetHealthBarVisible(playerid, bool:toggle)
 {
+	if (!s_EnableHealthBar[playerid]) 
```

Comment:
This is not what I meant, K&R is 
```
if (!s_EnableHealthBar[playerid]) {
    toggle = false;
}
```

Reply:
I don't understand why you doing that? What is your problem? I mean why you kidding me with that things?

Reply:
consistency is very important in programming. This is why style guides exist, to maintain a set of rules that a project uses. If standards and style guides are adhered to than the project overall becomes easily to understand.

Reply:
Do you maybe want like this:
if (!s_EnableHealthBar[playerid]) {toggle = false;}
Oscar i mean Slice, is not crazy like you, i did this function for players and servers who want this, and you kidding with that shits damn man

Reply:
Ye i agree with u, but if u want u can tell exactly what you want, i don't have time for 20 commits about small thing understand me please, i working i working every day, my job is similar like your and u want to learn me about programming okey

Reply:
It's good practice to adhere to consistent project guides/rules. Slice maybe did not enforce this before, but this is the proper way to program, and is what makes the overall project easier to understand and read as a whole.

If the project was full of different styles that PR's had, it would become much harder to read and might make people less likely to want to make a PR.

Reply:
I literally posted the exact code for you. ???

Reply:
I have my style you have your thats it, ask everyone about her style all programmers have style about the code, so, this not golang

Reply:
NO!!!! that is wrong. Never assume that. This is exactly why style guides exist, so people don't use their own style, and instead follow a proper set of guidelines.

Reply:
Aight man i will fix that so next time say on time to i write code on your style oke? 
The other things about this PR is okay or no?

Reply:
https://medium.com/level-up-web/what-is-a-programming-style-guide-and-why-should-you-care-9019e51bb7ad

Read that, it will explain exactly why style guides are used.

Reply:
Own style is if i working alone, if i work with team like this i post PR and i am in team okay, but why u don't before first comment said look zile u must use that style for this project, then will be okay all, but, my commit is okey, thats is k&r right? So what now its okay now to you accept pull or something is wrong with me or my code or what?

Reply:
I think we talk about bullshits let's finish this pr and on another all will be like your style i agree with that. I don't have anymore comments

Reply:
I shouldn't need to tell you this, you are the one making a PR, it is your job to look at the current script and try to copy the rules it uses. This is general knowledge for anyone working on other people's projects.

If you work on your own project, you can do what ever you want. But if you are working in a team, or you are working on an open source project (this), you adhere to the conventions and rules used by this project. 

This is so your PR's don't get rejected for not following the convention. If you had made a PR to one of Google's or Facebook's repos without following their guidelines, your PR would immediately be rejected, even if the code itself is good.

So please in future always follow the rules/conventions a repo already follows. Do not use your own style and do not assume there isn't one.

Reply:
Okay i said i will fix that is that okay? I agree with you.

Reply:
You now want to i rechange that line and another, add brackets?

Reply:
Yes, to keep consistency.

Reply:
sure bro, be here i will commit in few minutes i have free time now.